docs/configuration.md

```diff
@@ -51,7 +51,7 @@ Property
 **envoy-control.envoy.snapshot.localService.responseTimeout**                                          | Response timeout for localService                                                                                                                                                                           | 15s
 **envoy-control.envoy.snapshot.localService.connectionIdleTimeout**                                    | Connection idle timeout for localService                                                                                                                                                                    | 120s
 **envoy-control.envoy.snapshot.routes.status.enabled**                                                 | Enable status route                                                                                                                                                                                         | false
-**envoy-control.envoy.snapshot.routes.status.path-prefix**                                             | Path prefix of status route                                                                                                                                                                                      | /status
+**envoy-control.envoy.snapshot.routes.status.path-prefix**                                             | Path prefix of status route                                                                                                                                                                                 | /status
```

Comment:
what's the change? 🤔

Reply:
I haven't changed anything conciously, probably detekt or ktlint :|

Reply:
Tabs fixed:

![image](https://user-images.githubusercontent.com/2753650/89391944-23970a00-d709-11ea-929b-184754ddc19b.png)